README.md

```diff
@@ -52,20 +52,50 @@ Set up a development environment on any infrastructure, with a single command.
 * __All Ports__: The VPN connection enables access to all ports on the development environments, removing the need to setup port forwards over SSH connection.
 * __Works on my Machine__: Never experience it again.
 ## Quick Start
-To install Daytona all you need to do is execute this script:
+### Mac / Linux
+Install Daytona into /usr/local/bin
+```bash
+curl -sf -L https://download.daytona.io/daytona/get-server.sh | sudo bash
+```
+Or if you want to install Daytona to some other path where you don't need sudo
+```bash
+curl -sf -L https://download.daytona.io/daytona/get-server.sh | DAYTONA_PATH=/home/user/bin bash
+```
+### Windows
+<details>
+  <summary>AMD</summary>
+Run the following command in Windows PowerShell:
 
+```bash
```

Comment:
powershell is a supported language for GitHub code blocks

https://github.com/github-linguist/linguist/blob/4ac734c15a96f9e16fd12330d0cb8de82274f700/lib/linguist/languages.yml#L5392